Packs/Sixgill-Darkfeed/ReleaseNotes/1_1_5.md

```diff
@@ -0,0 +1,9 @@
+
+#### Playbooks
+##### Darkfeed Threat hunting-research
+- The playbook is using now the new script
+
+#### Scripts
+##### SixgillSearchIndicators
+- The old script ***SearchIndicators*** was deleted and the new ***SixgillSearchIndicators*** replaces it.
```

Comment:
instead of renaming the script (which means the old script will be deleted) let's deprecate the original and create a new copy with the updated name. this way we remove the risk of BC.